ddl/000009_export_current.down.sql

```diff
@@ -1,6 +1,6 @@
 BEGIN;
 
 DROP TABLE IF EXISTS catalog_branches_export_state;
-DROP TYPE catalog_branches_export_status;
+DROP TYPE If EXISTS catalog_branch_export_status;
```

Comment:
```suggestion
DROP TYPE IF EXISTS catalog_branch_export_status;
```
SQL is case-insensitive-ish in places like this, so it works, but it's not pretty.